packages/host/app/resources/realm-session.ts

```diff
@@ -58,11 +60,16 @@ export class RealmResource extends Resource<Args> {
     return this.token?.permissions?.includes('write');
   }
 
+  get realmToken() {
```

Comment:
Perhaps `rawRealmToken` is a better name for this getter. It tells you straight away that this is a raw string to be included in the header.